app/src/css.ts

```diff
@@ -256,6 +256,29 @@ function input() {
   return [base, dark, invertCalendarIcon]
 }
 
+function link() {
+  const createStyle = (selector: string, colorConfigPath: string, mode = <Mode>'light') => {
+    const style = new Style(`:is(.a, a)${selector}`, mode);
+    style.add('color', mode === 'light' ? l(colorConfigPath) : d(colorConfigPath));
+    style.add('cursor', 'pointer');
+    style.add('text-decoration', mode === 'light' ? l('link.textDecoration') : d('link.textDecoration'));
```

Comment:
You’re adding `text-decoration` and `cursor` to every state but it should only appear on the main `<a />` element.

Get what I mean?

Reply:
Yeah I get you, checkout the changes now